htdocs/product/class/product.class.php

```diff
@@ -620,6 +627,21 @@ class Product extends CommonObject
 	 * Service
 	 */
 	const TYPE_SERVICE = 1;
+	/**
+	 * Advanced feature: assembly kit
+	 */
+	const TYPE_ASSEMBLYKIT = 2;
```

Comment:
It seems this is not related to the feature stockable or not.
Thanks to split this other feature into a different PR.